test/__tests__/data/marketo_router_output.json

```diff
@@ -492,38 +509,38 @@
         "clientSecret": "3l3gJpzRsZagD6gu7tnTeKXz0bomLGnd",
         "trackAnonymousEvents": false,
         "createIfNotExist": true,
-        "customActivityEventMap": [
+        "rudderEventsMapping": [
           {
-            "from": "acq_signup_completed",
-            "to": "100026"
+            "event": "acq_signup_completed",
+            "marketoActivityId": "100026"
           },
           {
-            "from": "act_createwebinarform_submit",
-            "to": "100025"
+            "event": "act_createwebinarform_submit",
+            "marketoActivityId": "100025"
           },
           {
-            "from": "act_presentation_style",
-            "to": "100025"
+            "event": "act_presentation_style",
+            "marketoActivityId": "100025"
           },
           {
-            "from": "act_webinar_view",
-            "to": "100025"
+            "event": "act_webinar_view",
+            "marketoActivityId": "100025"
           },
           {
-            "from": "act_webinar_join",
-            "to": "100025"
+            "event": "act_webinar_join",
+            "marketoActivityId": "100025"
           },
           {
-            "from": "act_presentation_addteammember",
-            "to": "100025"
+            "event": "act_presentation_addteammember",
+            "marketoActivityId": "100025"
           },
           {
-            "from": "act_engagement_discussions_savediscussion",
-            "to": "100025"
+            "event": "act_engagement_discussions_savediscussion",
+            "marketoActivityId": "100025"
           },
           {
-            "to": "100025",
-            "from": "act_engagement_networking_savetime"
+            "event": "act_engagement_networking_savetime",
+            "marketoActivityId": "100025"
```

Comment:
Should we also similarly test the cases where `event` and `marketoPrimaryKey` are only present ?

Reply:
I don't think thats needed because it will have the same result as this one that is error thrown for required field for an event name